app/src/main/java/nextstep/shoppingcart/ui/screen/component/CartItemContainer.kt

```diff
@@ -0,0 +1,289 @@
+    ) {
+        Column {
+            CartItemTopSector(
+                title = cartItem.product.title,
+                onCartItemDelete = onCartItemDelete,
+            )
+            Spacer(modifier = Modifier.height(6.dp))
+            CartItemBottomSector(
+                imageUrl = cartItem.product.imageUrl,
+                price = cartItem.totalPrice,
+                count = cartItem.count,
+                onMinusCartItem = onMinusCartItem,
+                onPlusCartItem = onPlusCartItem,
+            )
+        }
+    }
+}
+
+@Composable
+private fun CartItemTopSector(
+    title: String,
+    onCartItemDelete: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier.fillMaxWidth(),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Text(
+            text = title,
+            fontWeight = FontWeight.W700,
+            fontSize = 20.sp,
+        )
+        CloseIconButton(
+            onClick = onCartItemDelete,
+            contentDescription = "Remove Button"
+        )
+    }
+}
+
+@Composable
+private fun CartItemBottomSector(
+    imageUrl: String,
+    price: Int,
+    count: Int,
+    onMinusCartItem: () -> Unit,
+    onPlusCartItem: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(IntrinsicSize.Max),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        ProductImage(
+            imageUrl = imageUrl,
+            contentDescription = "Cart Product Image",
+            modifier = modifier
+                .weight(1f)
+                .aspectRatio(136 / 84f)
+        )
+        Spacer(modifier = Modifier.width(26.dp))
+        CartItemInfo(
+            price = price,
+            count = count,
+            onMinusCartItem = onMinusCartItem,
+            onPlusCartItem = onPlusCartItem,
+            modifier = modifier
+                .weight(1f)
+                .padding(end = 10.dp)
+        )
+    }
+}
+
+@Composable
+private fun CartItemInfo(
+    price: Int,
+    count: Int,
+    onMinusCartItem: () -> Unit,
+    onPlusCartItem: () -> Unit,
+    formatter: MoneyFormatter = DefaultMoneyFormatter,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier.fillMaxHeight(),
+        horizontalAlignment = Alignment.End,
+        verticalArrangement = Arrangement.Bottom
+    ) {
+        Text(
+            text = "${formatter.format(price)}원",
+            fontSize = 16.sp,
+            fontWeight = FontWeight.W400
+        )
+        CartQuantitySector(
+            count = count,
+            onMinusCartItem = onMinusCartItem,
+            onPlusCartItem = onPlusCartItem,
+            modifier = Modifier.fillMaxWidth()
+        )
+    }
+}
+
+@Composable
+private fun CartQuantitySector(
+    count: Int,
+    onMinusCartItem: () -> Unit,
+    onPlusCartItem: () -> Unit,
```

Comment:
플러스카트 아이템을.... 하고 끝나는 느낌이 들어서요! Click이란 단어를 더 붙여보면 어떨까요?
```suggestion
    onMinusCartItemClick: () -> Unit,
    onPlusCartItemClick: () -> Unit,
```

Reply:
넵 바로 다음 스탭에서 수정해서 반영하겠습니다!